easybuild/tools/singularity/utilities.py

```diff
@@ -0,0 +1,467 @@
+# Copyright 2014-2017 Ghent University
```

Comment:
I've checkout this file and it's so badly formatted with mixed tabs and spaces. Can you please fix this?